packages/gestalt/src/contexts/DefaultLabelProvider.js

```diff
@@ -77,6 +82,11 @@ export const fallbackLabels: DefaultLabelContextType = {
     dismissConfirmationSecondaryActionText: 'No, go back',
     dismissConfirmationSecondaryActionTextLabel: 'No, go back to the overlay panel.',
   },
+  SheetMobile: {
+    accessibilityDismissButtonLabel: 'Close overlay panel',
+    accessibilityGrabberLabel: 'Grabber',
+    accessibilityLabel: 'Bottom sheet',
```

Comment:
Is this a useful label? Can we provide a useful default, or should this label be required?

Reply:
Yeah as per guidance in Accessibility. When it opens it reads, "Grabber, Bottom sheet" Bottom sheet can be customized.